protected/extensions/phpword/requestWord.php

```
@@ -0,0 +1,370 @@
+    private function addOtherServiceSection() {
+        $section = $this->phpWord->getLastSection();
+        $section->addText('3. OTHER SERVICE', ['bold' => true, 'size' => 9]);
+        // Content for this section is empty in the PDF, so keeping it minimal
+        $tableStyle = new Table();
+        $tableStyle->setBorderSize(6);
+        $tableStyle->setUnit(\PhpOffice\PhpWord\Style\Table::WIDTH_PERCENT);
+        $tableStyle->setWidth(100 * 100);
+        $table = $section->addTable($tableStyle);
+        $cellStyle = ['valign' => 'center', 'borderSize' => 6, 'borderColor' => '000000'];
+        $table->addRow()->addCell(null, $cellStyle); // Empty row for spacing
+        foreach($this->request->samps as $sample)
```

PHPMD (2.15.0)

Avoid unused local variables such as '$sample'. (Unused Code Rules)

(UnusedLocalVariable)